src/main/scala/com/datastax/gatling/plugin/response/DseResponseHandler.scala

```diff
@@ -88,13 +94,9 @@ abstract class DseResponseHandler[RS, Response <: DseResponse] extends StrictLog
     )
 
     stmt match {
-      case Some(gs: GraphStatement) =>
-        val unwrap = Try(
-          DseResponseHandler.sanitize(gs.unwrap(GraphProtocol.GRAPHSON_2_0).toString)
-        ).getOrElse(DseResponseHandler.sanitize(gs.unwrap(GraphProtocol.GRAPHSON_1_0).toString))
```

Comment:
This logic appears only to have existed to get to a Statement with a meaningful toString() method.  A SimpleGraphStatement, say, in 1.6.8 did not have a workable toString() method, but unwrapping to an underlying Statement impl seemed to get to a better spot.  I've confirmed that the relevant GraphStatement impls in 2.x _do_ have meaningful toString() methods... so the unwrap logic has been removed outright.